tests/interop/test/interop/ilp-client-interop-test.json

```diff
@@ -0,0 +1,1634 @@
+[
```

Comment:
We should not check-in this file directly, but use a git submodule instead. That's what we do in all client repos.